math/minuit/src/TLinearMinimizer.cxx

```diff
@@ -51,19 +51,18 @@ struct BasisFunction {
    const Func * fFunc;
 };
 
-
-//______________________________________________________________________________
-//
-//  TLinearMinimizer, simple class implementing the ROOT::Math::Minimizer interface using
-//  TLinearFitter.
-//  This class uses TLinearFitter to find directly (by solving a system of linear equations)
-//  the minimum of a
-//  least-square function which has a linear dependence in the fit parameters.
-//  This class is not used directly, but via the ROOT::Fitter class, when calling the
-//  LinearFit method. It is instantiates using the plug-in manager (plug-in name is "Linear")
-//
-//__________________________________________________________________________________________
-
+////////////////////////////////////////////////////////////////////////////////
+/// \class TLinearMinimizer
+/// \legacy{TLinearMinimizer, Consider switching to Minuit2.}
+///
```

Comment:
```suggestion
```
Since we should not deprecate the old Minuit, and this `TLinearMinimizer` is the ROOT-internal plugin to make the old Minuit available to the `ROOT::Fit::Fitter` interface, I would also not mark this as deprecated (in fact, the user should not have to read this documentation because it is an implementation detail).